UPGRADE-3.x.md

```diff
@@ -1,6 +1,10 @@
 UPGRADE 3.x
 ===========
 
+## Deprecated ModelsToArrayTransformer::$choiceList property
+
+ModelsToArrayTransformer::__construct() can be used with two parameters ($modelManager, $class) or three ($choiceList, $modelManager, $class )
```

Comment:
Yes there is : people calling the method with 3 params should call it with 2 (otherwise they will get a deprecation)